src/main/java/org/assertj/core/condition/MappedCondition.java

```diff
@@ -131,13 +135,33 @@ public boolean matches(FROM value) {
    * @return the mapped condition description .
    */
   protected String buildMappingDescription(FROM from, TO to) {
+  	return buildMappingDescription(from,to,true);
+  }
+
+  private String buildMappingDescription(FROM from, TO to,boolean withNested) {
     StringBuilder sb = new StringBuilder("mapped");
     if (!mappingDescription.isEmpty()) sb.append(format("%n   using: %s", mappingDescription));
     sb.append(format("%n   from: <%s> %s%n", from.getClass().getSimpleName(), from));
     sb.append(format("   to:   <%s> %s%n", to.getClass().getSimpleName(), from, to));
     sb.append("   then checked:");
-    sb.append(format("%n      %-10s", condition));
+    if(withNested) {
+    	sb.append(format("%n      %-10s", condition));
+    }
     return sb.toString();
   }
 
+  @Override
+  public Description conditionDescriptionWithStatus(FROM actual) {
+  	TO mappedObject = mapping.apply(actual);
+    Description descriptionsWithStatus =  condition.conditionDescriptionWithStatus(mappedObject);
+   String status= status(actual).label;
+   StringBuilder sb=new StringBuilder(status)
+   .append(" ")
+   .append(buildMappingDescription(actual,mappedObject,false));
+
+   List<Description> descriptions=new ArrayList<>();
+   descriptions.add(descriptionsWithStatus);
+    return new JoinDescription(sb.toString(),"]",descriptions);
+  }
```

Comment:
please format the code according to AssertJ convention

Reply:
formatted

Reply:
formatted

Reply:
this is not formatted with AssertJ code style which is mentioned in the [contributing guidelines](https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md).